CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
 ## jest 21.1
 
+* Breaking Change: Add Async Test Environment APIs (dispose is now teardown) ([#4506](https://github.com/facebook/jest/pull/4506))
```

Comment:
After you rebase, please move this up a bit to master changelog and follow new convention (v21.1 is already released, so this is not a proper place anyway)